quark/evaluator/pyeval.py

```diff
@@ -673,11 +673,11 @@ def _lookup_implement(self, instance_type, method_full_name, skip_self=False):
         ):
             next_class_pool.clear()
             for class_name in class_pool:
-                method = self.apkinfo.find_method(class_name, method_name, descriptor)
+                method_list = self.apkinfo.find_method(class_name, method_name, descriptor)
 
-                if method:
+                if method_list:
                     return PyEval.get_method_pattern(
-                        method.class_name, method.name, method.descriptor
+                        method_list[0].class_name, method_list[0].name, method_list[0].descriptor
```

Comment:
There are more than 79 characters in this line. Please consider inserting line breaks to make it shorter.